openstack/provision-hosts.yml

```diff
@@ -21,6 +21,7 @@
           - net-id: "{{ os_net_id }}"
         security_groups: "{{ security_group }}"
         state: "{{ nova_compute_state }}"
+        meta: "{{ host_metadata }}"
```

Comment:
Can you change this to have a default value so that it doesn't break for people reusing older dc1 files?     
``` 
meta: "{{ host_metadata | default() }}"
```

Otherwise, looks good to me!